src/is-object-visible/index.ts

```diff
@@ -0,0 +1,3 @@
+export function isObjectVisible(id: string): boolean {
+	return !(Game.getObjectById(id) === null);
```

Comment:
What's the reason for using this syntax instead of just checking for `!== null`?